tcp-transport/src/lib.rs

```diff
@@ -108,6 +111,8 @@ impl Transport for TcpConfig {
                 .map(|listener| {
                     // Pull out a stream of sockets for incoming connections
                     listener.incoming()
+                        .sleep_on_error(Duration::from_secs(1))
```

Comment:
The examples all use 100ms, why did you opt for waiting a full second on the connection that errored?

Reply:
I guess this should be in the `TcpConfig`.